src/libraries/System.Runtime/tests/System.Buffers.Tests/System.Buffers.Tests.csproj

```diff
@@ -5,6 +5,9 @@
     <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetOS)' == 'browser'">
+    <WasmXHarnessMaxParallelThreads>1</WasmXHarnessMaxParallelThreads>
```

Comment:
What was the problem with buffers ?

Reply:
OOM killed (from PR description)

```
[FAIL][8] System.Buffers.ArrayPool.Tests.ArrayPoolUnitTests.RentingSpecificLengthsYieldsExpectedLengths(requestedMinimum: 2097152, expectedLength: 2097152)
info: System.OutOfMemoryException : Out of memory
info:    at System.Buffers.ConfigurableArrayPool`1[[System.String, System.Private.CoreLib, Version=9.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].Rent(Int32 minimumLength)
info:    at System.Buffers.ArrayPool.Tests.ArrayPoolUnitTests.RentingSpecificLengthsYieldsExpectedLengths(Int32 requestedMinimum, Int32 expectedLength)
info:    at System.Object.InvokeStub_ArrayPoolUnitTests.RentingSpecificLengthsYieldsExpectedLengths(Object , Span`1 )
info:    at System.Reflection.MethodBaseInvoker.InvokeWithFewArgs(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
```

Reply:
We can re-parallelize them if you want to see the whole log

Reply:
It seems that GC is unable to keep up with the traffic. Let's create issue for that.

I wonder if Mono on x64 has the same problem but survives this because of more available memory ?

Reply:
Opened https://github.com/dotnet/runtime/issues/100628